utils/dataset_manifest/core.py

```diff
@@ -147,16 +150,18 @@ def __iter__(self):
 
 class DatasetImagesReader:
     def __init__(self,
```

Comment:
Could you please add typing for all arguments? For now it's looking a little bit strange....